graspologic/plot/plot.py

```diff
@@ -1397,6 +1397,8 @@ def networkplot(
         )
         ax.add_collection(lc)
         ax.set(xticks=[], yticks=[])
+        ax.xticks(x, " ")
```

Comment:
This should be 

```suggestion
        ax.set_xlabel("")
```
https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.set_xlabel.html

and likewise for y